03-first-aws-environment/bin/uniquify.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+#
+# Since we're creating an S3 bucket as part of this tutorial, we need the names of those
+# buckets to be unique, otherwise we'll run into issues with AWS S3 API errors blocking
+# our terraform applies. This scripts uniquifies all bucket names in the project.
+
+# Build random value
+RAND=$(cat /dev/urandom | tr -dc 'a-z0-9' | fold -w 12 | head -n 1)
+PROD_RAND=$(cat /dev/urandom | tr -dc 'a-z0-9' | fold -w 12 | head -n 1)
+
+# Updates our tfstate-backend's name to be unique
+yq eval --inplace \
+    ".components.terraform.tfstate-backend.vars.name = \"tfstate-$RAND\"" \
```

Comment:
Maybe we set an attribute instead?

Reply:
Will do depending on outcome of our conversation above.